liferay-tango/liferay-portal
# LPS-168143.Handle-error-parameter

Motivation
=======
For the Segmentation Initiative, we want to introduce a new error page for when an invalid operation occurs for created segments.
[Link to the task](https://issues.liferay.com/browse/LPS-170054)
[Link to the story](https://issues.liferay.com/browse/LPS-168143)

Proposed Solution
========
This PR contains only the Backend part for the ticket mentioned. It contains the logic to handle an exception when an invalid segmentation operation occurs. When the FF is true, the invalid operation the exception is handled with a variable "error" and send to the front. In case the FF is false the exception is thrown.

Steps to Verify:
----------------
For being able to verify this PR must be combined with the one with the frontend part: 
https://github.com/liferay-tango/liferay-portal/pull/3108
